inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/model/KnowledgeBase.java

```diff
@@ -382,7 +381,13 @@ public void applyMapping(KnowledgeBaseMapping aMapping)
         setPropertyTypeIri(aMapping.getPropertyTypeIri());
         setPropertyLabelIri(aMapping.getPropertyLabelIri());
         setPropertyDescriptionIri(aMapping.getPropertyDescriptionIri());
-        applyRootConcept(aMapping);
+        if (explicitlyDefinedRootConcepts.isEmpty()) {
```

Comment:
Because, when the user updates it.. we don't want to lost his set of explicitly defined root concept. Does it make sense or is there something I am missing ?

Reply:
Because, when the user updates it.. we don't want to lost his set of explicitly defined root concept. Does it make sense or is there something I am missing ?